tests/Unit/PrefixTest.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+use Jitterbug\Models\Format;
+use Jitterbug\Models\Prefix;
+use Jitterbug\Models\Collection;
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use Illuminate\Foundation\Testing\DatabaseTransactions;
+
+class PrefixTest extends TestCase
+{
+  use DatabaseTransactions;
+
+  public function testFindPrefixLabel()
+  {
+    $this->disableExceptionHandling();
+    $collection = factory(Collection::class)->create(['collection_type_id' => 3]);
+    $prefix1 = factory(Prefix::class)->create(['deleted_at' => null, 'collection_type_id' => 3]);
+    $prefix2 = factory(Prefix::class)->create(['deleted_at' => null, 'collection_type_id' => 4]);
+    $format1 = factory(Format::class)->create();
+    $format2 = factory(Format::class)->create();
+
+    // attach the formats to the prefix
+    $format1->prefixes()->attach($prefix1->id);
```

Comment:
Add a couple of tests checking for expected exceptions for invalid data after earlier code is removed?